packages/world/mud.config.mts

```diff
@@ -105,6 +105,12 @@ export default mudConfig({
       tableIdArgument: true,
       storeArgument: true,
     },
+    UniqueEntity: {
+      directory: "/modules/uniqueentity/tables",
+      primaryKeys: {},
+      schema: "uint256",
```

Comment:
do we expect entity IDs to be uint256 vs bytes32? I had started using bytes32 in my v2 emojimon project (since it more closely matches a key)

Reply:
ultimately I wanna try some built-in user-defined `EntityId` as the type.
I also want to allow default metaconfig, so users can choose what their default keys are.
For now whatever is fine for me, should I switch to bytes32?

Reply:
I agree with changing to bytes32 for now to more closely match a key